pkg/api/client.go

```diff
@@ -67,27 +67,36 @@ func (c *Client) getClientsWithFilters(filtersEnabled bool, mac string) ([]Netwo
 	setHeaders(req, c.token)
 	resp, err := c.httpClient.Do(req)
 	if err != nil {
-		return nil, err
+		return nil, nil, err
 	}
 
 	defer resp.Body.Close()
 	body, err := io.ReadAll(resp.Body)
 	if err != nil {
-		return nil, err
+		return nil, nil, err
 	}
 	log.Debug().Bytes("data", body).Msg("Received data from clients endpoint")
 
 	clientdata := clientResponse{}
 	err = json.Unmarshal(body, &clientdata)
 
-	return clientdata.Result.Data, err
+	return clientdata.Result.Data, &clientdata.Result.Stats, err
 }
 
 type clientResponse struct {
 	Result data `json:"result"`
 }
 type data struct {
-	Data []NetworkClient `json:"data"`
+	Data 		[]NetworkClient `json:"data"`
+	Stats		ClientStats		 `json:"clientStat"`
```

Comment:
We shouldn't need to returns `stats` at all from the API as all the data we need to compute the client totals is already available under each `NetworkClient`. (See the comments a bit further down)